MailCore/Models/Message.swift

```diff
@@ -347,6 +361,7 @@ public final class Message: Object, Decodable, ObjectKeyIdentifiable {
         snoozeState = try? values.decodeIfPresent(SnoozeState.self, forKey: .snoozeState)
         snoozeAction = try values.decodeIfPresent(String.self, forKey: .snoozeAction)
         snoozeEndDate = try values.decodeIfPresent(Date.self, forKey: .snoozeEndDate)
+        headers = try values.decodeIfPresent(MessageHeaders.self, forKey: .headers)
```

Comment:
This can be optional. Just in case something changes with the api
```suggestion
        headers = try? values.decodeIfPresent(MessageHeaders.self, forKey: .headers)
```